substrate/frame/support/src/traits/tokens/fungible/mod.rs

```diff
@@ -58,8 +58,9 @@
 //!   3 holds for 100 units, the account can spend its funds for any reason down to 300 units, at
 //!   which point the holds will start to come into play.
 //!
-//! - **Frozen Balance**: A freeze on a specified amount of an account's free balance until a
-//!   specified block number.
+//! - **Frozen Balance**: A freeze on a specified amount of an account's balance. There's no 
+//!   restriction preventing an account from freezing over its available balance. This amount still
+//!   belongs to the account holder, but cannot be spent.
```

Comment:
>This amount still belongs to the account holder, but cannot be spent.

Can you be more specific please? Since the frozen balance can be over the total balance, it does not belong to the account. If an account has 10 total balance and 20 frozen, the 20 does not belong to the account, just the lock does.

Reply:
fair point

Reply:
Definitely! Let me rephrase it then :) Should I close this PR and create a new one ?

Reply:
> Should I close this PR and create a new one ?

def no, just make the fix and commit the changes

Reply:
@shawntabrizi  @ggwpez  done! I hope now it's better explained :)